index.bs

```diff
@@ -124,8 +124,9 @@ url: https://aomediacodec.github.io/av1-spec/av1-spec.pdf#; spec: AV1; type: dfn
     text: Sequence Header OBU
     text: Metadata OBU
     text: Temporal Unit
-
-url: http://aomediacodec.github.io/av1-spec/av1-spec.pdf#; spec: AV1; type: dfn;
```

Comment:
Stray change?

Reply:
No. This line is (almost) a duplicate of line 122, except that it uses http instead of https. I think initially we had 2 sections to distinguish AV1-defined words (e.g. Temporal Unit) from AV1-defined keywords (e.g. mono_chrome). I'm not sure we need to make this distinction. WDYT?